INSTALL.md

```diff
@@ -2,7 +2,15 @@
 
 Go to [https://takeout.waffle.io](https://takeout.waffle.io) for more information on pricing and to sign up for a free 45 day trial.
 
-Getting Started guides and other Waffle.io documentation can be found [here](https://github.com/waffleio/waffle.io/wiki).
+Getting Started guides and other Waffle.io documentation can be found [here](https://help.waffle.io).
+
+## Installation Types
+* [Bring Your Own
+  Server](https://github.com/waffleio/waffle.io-takeout/blob/master/INSTALL.md#installation)
```

Comment:
not that it is terribly important, but I think you can make the link just`#installation` and then it would be relative so the link works in your PR as well.